SurfaceDevCenterManager/Program.cs

```diff
@@ -769,6 +784,52 @@ private static async Task<ErrorCodes> MainAsync(string[] args)
                     }
                 }
             }
+            else if (TranslateOption)
+            {
+                Console.WriteLine("> Translate Option");
+
```

Comment:
Perhaps, set **reval = 0**  here, so that a previously assigned value isn't accidentally used below. 
(If all the IDs are non-NULL, retval does not get assigned here and will use earlier value set above.)

Reply:
I am trying to leave it simplified in that whatever 'major' operation is requested is where execution falls in the if/if else and that one will end up processing the rest which means only one will need to handle retval